lib/node_modules/@stdlib/assert/is-boolean-array/examples/index.js

```diff
@@ -13,11 +13,10 @@
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
-* limitations under the License.
+* limitations under t
+ License.
```

Comment:
```suggestion
* limitations under the License.
```

Please avoid making extraneous changes. When submitting a PR, it is always best practice to double-check your work and perform a self-review.